content/en/docs/Installation/freebsd.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "FreeBSD Install"
+linkTitle: "FreeBSD"
+date: 2020-03-14
```

Comment:
Docsy sorts the menu by this date. Please change it so "FreeBSD" shows up after "macOS". You can see the results of your PR here: https://deploy-preview-32--navidrome.netlify.app/docs/installation/

Reply:
Interesting...  If I am understanding this correctly, in order to get things arranged correctly, I need to change the macOS date to 2017-01-04 and set FreeBSD's to 2017-01-03?